encoding/ccf/encode_type.go

```diff
@@ -318,6 +318,13 @@ func (e *Encoder) encodeReferenceType(
 	)
 }
 
+func (e *Encoder) encodeAuthorization(
+	auth cadence.Authorization,
+) error {
+	// TODO: implement this
```

Comment:
Is this implemented in a subsequent PR?

Reply:
No, I spoke to @fxamacker about this and added this TODO (and created an issue) for her to track it; she wants to implement this herself in the future.

Reply:
No, I spoke to @fxamacker about this and added this TODO (and created an issue) for her to track it; she wants to implement this herself in the future.